doc/md/fundamentals/7-modules-imports.md

```diff
@@ -26,10 +26,10 @@ Debug.print("Hello, world!");
 
 The `mo:` prefix identifies a Motoko module. The declaration does not include the `.mo` file extension.
 
-You can also selectively import and rename a subset of named values from a module by using the object pattern syntax:
+You can also selectively import and rename a subset of named values and types from a module by using the object pattern syntax:
 
 ``` motoko
-import { map; find; foldLeft = fold } = "mo:base/Array";
+import { type Map; get; contains; foldLeft = fold } = "mo:base/OrderedMap";
```

Comment:
This example isn't correct actually (try cd doc; make preview). I tried fixing it using List instead but then uncovered a bug. Perhaps the interpreters are still buggy?

See https://github.com/dfinity/motoko/pull/5362 for a repro.

Reply:
```suggestion
import { type List; get; foldLeft = fold } = "mo:base/List";
```
might actually type check after bug fix.